tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/documentation/MarkdownDocumentationTest.java

```diff
@@ -319,6 +319,11 @@ public void testDocFunction() {
         Assert.assertEquals(references.get(7).type, DocumentationReferenceType.ANNOTATION);
         Assert.assertEquals(references.get(7).referenceName, "annot");
 
+        var fooFunction = packageNode.getFunctions().get(5).getMarkdownDocumentationAttachment();
+        var exapmpleParam = fooFunction.getParameterDocumentations().get("example");
```

Comment:
Why we use `var` instead of exact types?

Reply:
It is okay to use var right.

Reply:
Even in Ballerina, we don't encourage to use `var`, because it reduces the readability of the code.

Reply:
I replaced var with `BLangMarkdownParameterDocumentation`.